js/deer-utils.js

```diff
@@ -153,7 +153,25 @@ export default {
                                         continue Leaf;
                                     }
                                     else {
-                                        obj = Object.assign(obj, val);
+                                        // Assign this to the main object.
+                                        if(obj[k]) {
+                                            // It may be already there as an Array with some various labels
+                                            if (Array.isArray(obj[k])){
+                                                let deepMatch = false
+                                                for(const e of obj[k]) {
+                                                    if(e.name===val.name){
```

Comment:
I'm honestly not sure...
This use is rigging that is within the Web Annotation standard, but extended for our expectations, so having some API-style requirement isn't crazy, but there are reasons it may not be. I mean
` if ( name, label, title, dc:title, key, rdf:label, schema:name, alternateTitle, alias )` is clearly too much, but I'm not sure where the use will be.